src/lib/device-online-state.ts

```diff
@@ -157,16 +202,35 @@ class DeviceOnlineStateManager {
 				}
 			})
 			.then(() => {
+				this.queueStatsInterval = setInterval(async () => {
```

Comment:
There's no need to actually assign this anywhere, you can do `setInterval(...).unref()`